libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -565,8 +565,9 @@ void AP_DDS_Client::on_topic(uxrSession* uxr_session, uxrObjectId object_id, uin
 
             for (uint8_t i = 0; i < MIN(8U, rx_joy_topic.axes_size); i++) {
                 // Per MAVLink spec a value of UINT16_MAX means to ignore this field. Setting the
-                // channel override to 0U disable the overrides and releases the channel back to RC
-                if (static_cast<uint16_t>(rx_joy_topic.axes[i])!= UINT16_MAX) {
+                // channel override to 0U releases the channel back to the RC.
+                // As we are dealing with floats, we check that the value is below UINT16_MAX.
```

Comment:
Yea, seems right. Or, we could also just use NaN to mean invalid in ROS if it's documented. The UINT16_MAX is only because of mavlink using int's and not having NaN on that message, which shouldn't really affect this interface if there's better ways.

Reply:
I replaced it with a Nan and tested on the SITL.